src/modules/api.ts

```diff
@@ -154,29 +153,32 @@ export async function fetchContract(address, tokenContracts) {
     promises[version].linear(),
   ])
 
+  const formattedDecimals = Number(decimals)
+
   const contract = {
     version,
     symbol,
     address,
-    balance: parseInt(balance, 10) / 10 ** decimals,
+    balance: Number(balance) / 10 ** formattedDecimals,
```

Comment:
Maybe we could use the same parsing fn for all of this.
It's also used  for the release logs:

```typescript
const cumulative = Big(Number(cumulativeReleased) || 0).div(Big(10) ** Big(decimals))
```